test/converters/xcresult3/xcresulttool.go

```diff
@@ -75,13 +81,24 @@ func xcresulttoolGet(xcresultPth, id string, v interface{}) error {
 
 // xcresulttoolExport exports a file with the given id at the given output path.
 func xcresulttoolExport(xcresultPth, id, outputPth string) error {
-	args := []string{"xcresulttool", "export", "--path", xcresultPth, "--id", id, "--output-path", outputPth, "--type", "file"}
-	isLegacyFlag, err := isLegacyFlagNeededForXcresulttoolVersion()
+	args := []string{"xcresulttool", "export"}
+
+	supportsNewMethod, err := supportsNewExtractionMethods()
 	if err != nil {
 		return err
 	}
-	if isLegacyFlag {
-		args = append(args, "--legacy")
```

Comment:
Don't we use the `--legacy` flag anymore, not even with the legacy exporter?

Reply:
Reintroduced the `legacy` flag as we need it fallback extraction.